src/iqm/iqm_client/iqm_client.py

```diff
@@ -331,23 +334,25 @@ def _validate_circuit_instructions(
         architecture: QuantumArchitectureSpecification,
         circuits: CircuitBatch,
         qubit_mapping: Optional[dict[str, str]] = None,
+        validate_moves: MoveGateValidationMode = MoveGateValidationMode.STRICT,
     ):
         """Validates that the instructions target correct qubits in the given circuits.
 
         Args:
-          architecture: the quantum architecture to check against
-          circuits: list of circuits to be checked
-          qubit_mapping: Mapping of logical qubit names to physical qubit names.
-              Can be set to ``None`` if all ``circuits`` already use physical qubit names.
-              Note that the ``qubit_mapping`` is used for all ``circuits``.
-
+            architecture: the quantum architecture to check against
+            circuits: list of circuits to be checked
+            qubit_mapping: Mapping of logical qubit names to physical qubit names.
+                Can be set to ``None`` if all ``circuits`` already use physical qubit names.
+                Note that the ``qubit_mapping`` is used for all ``circuits``.
+            validate_moves: Option for bypassing full or partial MOVE gate validation as described in
+                :class:`MoveGateValidationMode`.
```

Comment:
```suggestion
            validate_moves: Option for bypassing full or partial MOVE gate validation.
```
Types should not be repeated in the docstring since they're in the annotation anyway, and the Sphinx docs will render them as links.